examples/cpu_render.rs

```diff
@@ -137,15 +135,17 @@ fn main() {
             }
         }
 
-        use show_image::{ImageInfo, ImageView};
-        let binding = img.into_raw();
-        let image = ImageView::new(
-            ImageInfo::rgb8(viewport_size_width, viewport_size_height),
-            &binding,
-        );
-
-        // Create a window with default options and display the image.
-        window.set_image("image-001", image).ok().unwrap();
+        img.save("example_junk_cpu_render.png").ok().unwrap();
+        std::process::exit(0);
+        // use show_image::{ImageInfo, ImageView};
```

Comment:
Dead code still needed?

Reply:
It's going to be reverted, sorry about this!